voidcrew/code/game/objects/noose.dm

```diff
@@ -0,0 +1,150 @@
+/obj/item/stack/cable_coil/building_checks(datum/stack_recipe/noose_recipe, multiplier)
+	if(noose_recipe.result_type == /obj/structure/chair/noose)
+		if(!(locate(/obj/structure/chair) in get_turf(usr)))
+			to_chat(usr, "<span class='warning'>You have to be standing on top of a chair to make a noose!</span>")
+			return FALSE
+	return ..()
+
+/obj/structure/chair/noose //It's a "chair".
+	name = "noose"
+	desc = "Well this just got a whole lot more morbid."
+	icon_state = "noose"
+	icon = 'voidcrew/icons/obj/objects.dmi'
+	layer = FLY_LAYER
+	flags_1 = NODECONSTRUCT_1
+	///So that the noose sprite can overlay the person beind hanged.
+	var/mutable_appearance/overlay
+
+/obj/structure/chair/noose/wirecutter_act(mob/user, params)
+	. = ..()
+	user.visible_message("[user] cuts the noose.", "<span class='notice'>You cut the noose.</span>")
+	if(has_buckled_mobs())
+		for(var/mob/living/buckled_mob in buckled_mobs)
+			if(buckled_mob.has_gravity())
+				buckled_mob.visible_message("<span class='danger'>[buckled_mob] falls over and hits the ground!</span>")
+				to_chat(buckled_mob, "<span class='userdanger'>You fall over and hit the ground!</span>")
+				buckled_mob.Knockdown(60)
+				buckled_mob.adjustBruteLoss(10)
+	var/obj/item/stack/cable_coil/dropped_cable = new(get_turf(src))
+	dropped_cable.amount = 25
+	qdel(src)
+	return
+		
+/obj/structure/chair/noose/Initialize()
+	. = ..()
+	pixel_y += 16 //Noose looks like it's "hanging" in the air
+	overlay = image(icon, "noose_overlay")
+	overlay.layer = FLY_LAYER
+	add_overlay(overlay)
+
+/obj/structure/chair/noose/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/structure/chair/noose/post_buckle_mob(mob/living/buckled_mob)
+	if(has_buckled_mobs())
+		src.layer = MOB_LAYER
+		START_PROCESSING(SSobj, src)
+		buckled_mob.dir = SOUTH
+		animate(buckled_mob, pixel_y = initial(pixel_y) + 8, time = 8, easing = LINEAR_EASING)
+	else
+		layer = initial(layer)
+		STOP_PROCESSING(SSobj, src)
+		buckled_mob.pixel_x = initial(buckled_mob.pixel_x)
+		pixel_x = initial(pixel_x)
+		buckled_mob.pixel_y = buckled_mob.get_standard_pixel_y_offset(buckled_mob.lying_angle)
+
+/obj/structure/chair/noose/user_unbuckle_mob(mob/living/target_mob,mob/living/user)
+	if(has_buckled_mobs())
```

Comment:
everything in this proc only passes if `has_buckled_mobs()` is true, just make this an early return